src/main/java/eiteam/esteemedinnovation/commons/handler/GenericTickHandler.java

```diff
@@ -51,7 +50,7 @@
 
     static {
         if (FMLCommonHandler.instance().getSide() == Side.CLIENT) {
-            FMLLog.info("[EI] Getting some fields from reflection for Tick Handling.");
+            EsteemedInnovation.logger.info("[EI] Getting some fields from reflection for Tick Handling.");
             itemInMainHandField = ReflectionHelper.getField("itemStackMainHand", "field_187467_d", ItemRenderer.class);
```

Comment:
Note (mostly to myself):
* Double check that these fields are correct
* Go through all ReflectionHelper usages